cookbooks/upstage/financial_analysis.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    doc_base = []
</pre><p><code>doc_base</code> is irrelevant in this code. Maybe move it to the next cell? </p><p>Also, for clarity, maybe rename it to something like <code>extracted_texts</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UpstageAI/cookbook/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>